docs/tools/checkout/README.md

```diff
@@ -0,0 +1,42 @@
+---
+id: Checkout
+title: Checkout
+description: >-
+  Explainer on the what the checkout endpoint is which exists as part of
+  the Unlock Protocol app.
+---
+
+# Checkout
+
+Unlock Protocol has an app, most people are familiar with the Dashboard, Members
+and Keychain portions, but that app has an endpoint called checkout which accepts
+a configuration object as params and returns a custom configured checkout experience.
+
+## The experience
+This experience can also be used in a few different ways:
+1. To implement an Unlock enabled sign-in with Ethereum ([SIWE](../sign-in-with-ethereum/))
+1. Stand-alone as a link you distribute to your members via email, Discord, etc.
```

Comment:
I wonder about numbers here :D

Reply:
Believe it or not this works in md and then you never have to worry about changing the numbers when you change the list.
<img width="292" alt="Screen Shot 2022-09-01 at 3 11 29 PM" src="https://user-images.githubusercontent.com/18197239/188022065-df3da82f-9796-4a8e-b7ad-09bd3dbcc382.png">